common/persistence/sql/sqlplugin/mysql/namespace.go

```diff
@@ -33,38 +33,38 @@ import (
 
 const (
 	createNamespaceQuery = `INSERT INTO 
- namespaces (id, name, is_global, data, data_encoding, notification_version)
- VALUES(?, ?, ?, ?, ?, ?)`
+ namespaces (partition_id, id, name, is_global, data, data_encoding, notification_version)
+ VALUES(?, ?, ?, ?, ?, ?, ?)`
 
 	updateNamespaceQuery = `UPDATE namespaces 
  SET name = ?, data = ?, data_encoding = ?, notification_version = ?
- WHERE shard_id=54321 AND id = ?`
+ WHERE partition_id=54321 AND id = ?`
 
 	getNamespacePart = `SELECT id, name, is_global, data, data_encoding, notification_version FROM namespaces`
 
-	getNamespaceByIDQuery   = getNamespacePart + ` WHERE shard_id=? AND id = ?`
-	getNamespaceByNameQuery = getNamespacePart + ` WHERE shard_id=? AND name = ?`
+	getNamespaceByIDQuery   = getNamespacePart + ` WHERE partition_id=? AND id = ?`
+	getNamespaceByNameQuery = getNamespacePart + ` WHERE partition_id=? AND name = ?`
 
-	listNamespacesQuery      = getNamespacePart + ` WHERE shard_id=? ORDER BY id LIMIT ?`
-	listNamespacesRangeQuery = getNamespacePart + ` WHERE shard_id=? AND id > ? ORDER BY id LIMIT ?`
+	listNamespacesQuery      = getNamespacePart + ` WHERE partition_id=? ORDER BY id LIMIT ?`
+	listNamespacesRangeQuery = getNamespacePart + ` WHERE partition_id=? AND id > ? ORDER BY id LIMIT ?`
 
-	deleteNamespaceByIDQuery   = `DELETE FROM namespaces WHERE shard_id=? AND id = ?`
-	deleteNamespaceByNameQuery = `DELETE FROM namespaces WHERE shard_id=? AND name = ?`
+	deleteNamespaceByIDQuery   = `DELETE FROM namespaces WHERE partition_id=? AND id = ?`
+	deleteNamespaceByNameQuery = `DELETE FROM namespaces WHERE partition_id=? AND name = ?`
 
-	getNamespaceMetadataQuery    = `SELECT notification_version FROM namespace_metadata`
-	lockNamespaceMetadataQuery   = `SELECT notification_version FROM namespace_metadata FOR UPDATE`
-	updateNamespaceMetadataQuery = `UPDATE namespace_metadata SET notification_version = ? WHERE notification_version = ?`
+	getNamespaceMetadataQuery    = `SELECT notification_version FROM namespace_metadata WHERE partition_id = 54321`
+	lockNamespaceMetadataQuery   = `SELECT notification_version FROM namespace_metadata WHERE partition_id = 54321 FOR UPDATE`
+	updateNamespaceMetadataQuery = `UPDATE namespace_metadata SET notification_version = ? WHERE notification_version = ? AND partition_id = 54321`
 )
 
 const (
-	shardID = 54321
+	partitionId = 54321
```

Comment:
nit:
```suggestion
	partitionID = 54321
```

Reply:
Where does this number come from? I see it was default value in schama before.

Reply:
Nowhere other than schema/current usage, just a constant value that is used to pin to a DB shard